src/game/stage/results.c

```diff
@@ -68,7 +68,7 @@ static const u16 sStageScoreBonusesTexts[3][3] = {
     { 26, SA2_ANIM_SCORE_BONUSES, SA2_ANIM_VARIANT_SCORE_BONUSES_SP_RING },
 };
 
-const u16 gUnknown_080D71CC[3] = { (DISPLAY_WIDTH / 2) - 120, (DISPLAY_WIDTH / 2) - 51, (DISPLAY_WIDTH / 2) + 53 };
+const u16 sStageResultsTextOffset[3] = { 0, 69, 173 };
```

Comment:
Since you correctly chose a variable name starting with s, you could also make it a static variable:
static const u16

Reply:
Sure